html/index.html

```diff
@@ -36,10 +36,38 @@
   }
 }
     </script>
+    <script src="./js/jquery.min.js"></script>
     <!--#include file="../ssi/common_head_back.include" -->
     <!--#include file="../data/ssi/custom_head.include" -->
   </head>
 
+  <script>
+    $(document).ready(function(){
+      var timezoneOffset=new Date().getTimezoneOffset();
+      var xhttp = new XMLHttpRequest();
+      $('#query').keyup(function()
+      {
+        xhttp.abort();
+        var q=$(this).val();
+
+        if (q.length==0) {
+          $("#liveResponse").text("");
+        }
+        else{
+              xhttp.onreadystatechange = function() {
+              if (this.readyState == 4 && this.status == 200) {
+               console.log(JSON.parse(this.responseText).answers[0].actions[0].expression);
+               $("#liveResponse").text(JSON.parse(this.responseText).answers[0].actions[0].expression);
+              }
+            };
+            xhttp.open("GET", "/susi/chat.json?timezoneOffset="+timezoneOffset+"&q="+q, true);
+            xhttp.send();
+        }
+
+      })
+    })
```

Comment:
When you've included `jquery` why are you still using vanilla js style of creating an XHR request explicitly? This can be done by `getJSON()` and with other related jQuery based Ajax implementations.

Reply:
There is an onchange listener on the text field which creates a new request and deletes the previous one when the change occurs in the input field.

Reply:
Sure but I fail to understand why this is still written the way it is here. Why mix JS and jQuery? The onchange listeners and so on can also be done with jQuery and doesn't really need javascript in its vanilla version. Also, maybe please break the relevant js into a separate script file?